extensions/sql-database-projects/src/models/project.ts

```diff
@@ -179,6 +179,22 @@ export class Project {
 		return fileEntry;
 	}
 
+	public async deleteFileFolder(entry: ProjectEntry): Promise<void> {
+		// compile a list of folder contents to delete; if entry is a file, contents will contain only itself
```

Comment:
So... as per VS/SSDT behavior - we shouldnt do that. We should call the button remove from project and only remove it from project not delete from disk.

Reply:
Ah, this is "Exclude from Project", not "Delete"?  Both options exist in SSDT

![image](https://user-images.githubusercontent.com/1609827/85476801-e5d18e00-b56d-11ea-91e6-06a85e7c081d.png)

Reply:
I left Delete as is because if the user just uses the Explorer delete, it wouldn't update the sqlproj, but also added an Exclude option to match SSDT.

Reply:
ah didnt ever relalize SSDt has delete too. Makes sense then. Please make sure that the pop up confirmation explicitly says that **the file will be deleted from disk**